### ISSUE TITLE
[MU4 Issue] User cannot adust object with keyboard arrows if "Appearance" is open (regression)

### ISSUE THREAD

**Issue by rgreen5** [musescore/MuseScore#13627]
**Describe the bug**

In MS3 you could adjust the position of a text object with the keyboard arrows while monitoring the exact offset value in the Inspector.

However, MS4 does not allow you to do this. If you open the "Appearance" box, you cannot return the focus to the selected score object without closing the "Appearance" box. IMV, a significant regression.

There needs to be a way for "Appearance" to stay open _and_ allow the use of keyboard arrows to move the object _directly_ in the score.

**To Reproduce**

1. Open the attached score in MS3 and MS4. 
[position_adjust_issue.zip](https://github.com/musescore/MuseScore/files/9715347/position_adjust_issue.zip)
2. In MS3, verify that you can click on the staff text and adjust position with the keyboard arrows while monitoring the exact offset values in the Inspector.
3. Now try to do the same in MS4.

RESULT: You cant.

**Expected behavior**

The user should be able to reposition the text object directly with the keyboard arrows and be able to see the Offset values (as was possible in MS3) in "Appearance".

**Platform information**

OS: Linux Mint 20.1, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-3187285402, revision: github-musescore-musescore-7857d7f

**Comment by rgreen5** [comment]
This is a significant regression in usabilty from MS3. Contrast the behaviour with MS4:

In MS3, you just click on the object and the Inspector opens displaying the numerical values of X and y offset values in the **Element** section.


Not only is the focus lost, in error, from the selected element (after opening **Appearance**), but restoring the focus is awkward. If you click on the element again  it goes into edit mode and you cannot use the arrows to move the object. You have to click on the object, then press Escape.

So to recap. You have to (1) click on the object, (2) click on **Appearance** (3) click on the object again and (4) press Escape, before you can use the arrow keys to move the object (and. That's 4 steps to do the most simple thing. What a palaver!






_That's 1 Step to do in MS3 what it takes 4 steps in MS4_!

**Comment by MarcSabatella** [comment]
I'm not understanding.  Why would you click on the object again after clicking Appearance, or press Escape?  For me the spin boxes work perfectly right away,  sin't that the whole point of going to the properties panel - to actually use the spinboxes?  Having to perform an extra operation just to move focus to the panel I just opened for the purpsoe of using the spinboxes would make no sense.

I agree it's a drag to need to open Appearance just to access those spinboxes, but aside from that I'm not seeing any other regression here.

**Comment by rgreen5** [comment]
> Isn't that the whole point of going to the properties panel - to actually use the spinboxes?

Or you may wish to use the keyboard arrow keys to move the object to an exact position by using the offset counter reading?

**Comment by MarcSabatella** [comment]
You might indeed, but I suspect that’s less common than than actually using the spin boxes for that sort of precision.  Plus of course you might want keyboard of the *other* controls.   It would be bad to make the more common use cases harder.

Having a way to *show* the offset without interfering with keyboard focus would be preferable to breaking normal keyboard navigations.

**Comment by rgreen5** [comment]
> Having a way to _show_ the offset without interfering with keyboard focus would be preferable to breaking normal keyboard navigations.

That would suit me.

**Comment by Eism** [comment]
I think you are wrong. 

Our navigation system works in such a way that when a popup(or dialog) is opened, all focus is transferred to this popup. Because of it, the user can start navigating inside the popup by clicking any arrow(even if no element inside the popup has received focus). 
It seems to me that this is more understandable to the user than the fact that when the pop-up is open, the arrows will navigate through the elements inside the score, but when Escape is pressed, the pop-up should close.

**Comment by rgreen5** [comment]
Hopefully the new title will make the issue clearer.

**Comment by oMrSmith** [comment]
I'm not saying this with any anger, but popups inside an inspector get me absolutely lost,
especially if the little arrow doesn't even point to where the pop up originates from:

![image](https://user-images.githubusercontent.com/78870770/218870553-0111eb65-fee9-4d1b-ad2e-e4fceea67bf0.png)

I think drop down menus, would have been the better solution.

**Comment by oMrSmith** [comment]
https://user-images.githubusercontent.com/78870770/218873252-a19d2fed-c8af-4043-bdc3-379eb02651d8.mp4

**Comment by MarcSabatella** [comment]
What version is that?  Neither 4.0.1 nor current 4.0.2 nightlies have an arrow like that for staff text properties; it's a simple button that opens a separate dialog.

**Comment by SteveBlower** [comment]
I think this is related #12842

**Comment by oMrSmith** [comment]
Marc, the arrow is not a button, it is a visual indicator, supposed to point to the button, which opens the pop-up.

**Comment by MarcSabatella** [comment]
What I am saying is, I don't see an arrow at all.  I only see a button.  There also seems to be a typo in your version there ("ext" instead of "text".  Arte you sure this is an actual released version of MuseScore?

Here's how it looks for me:

![image](https://user-images.githubusercontent.com/1799936/219060878-5cdcbc11-18e1-4a92-94da-a6e807e76aca.png)

Even if I change to the GB language as it seems you are using, I don't get those errors:

![image](https://user-images.githubusercontent.com/1799936/219061441-dacb25ae-f8ed-44bd-a385-936ca833396c.png)

**Comment by MarcSabatella** [comment]
Anyhow, this also seems entirely unrelated to the issue.  I recommend starting a thread on the Support forum to see if anyone else can reproduce these strange behaviors you are seeing.

**Comment by oMrSmith** [comment]
> What I am saying is, I don't see an arrow at all.

That's super strange. Just to be sure I downloaded the last nighlty, even reset my preferences
and still: I'm seeing what, that you don't see.
